### PR TITLE
Feat/orders management

### PR DESCRIPTION
**Pull request by Parle648**
## Describe your changes

- Create second stage page UI + Connect Fe to Be. At this page you can see only orders without route

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://codecraftersintership.atlassian.net/browse/SCRUM-43)

## Loom video
  [Loom Video](https://www.loom.com/share/7db795d20e8e45c9b2804b32f53c1fbb?sid=dfb77a7d-b006-42bf-b704-ef97c148fb0e)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [x] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [x] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/pages/CreateRoute/SecondStage/components/OrderManagementCard/styles.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  display: 'none',
+  ...FONT,
+};
+
+export const luggageStyles: SxProps<Theme> = {
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  '& .MuiTypography-root': {
+    fontSize: '14px',
```
Comment (lyudad) [inline src/pages/CreateRoute/SecondStage/components/OrderManagementCard/styles.ts]: 1.3.2

File: src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/index.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <Box sx={rowStyles}>
+      <FormProvider {...methods}>
+        <Box sx={formWrapper}>
+          <img
+            style={{ cursor: 'pointer' }}
+            src={searchIcon}
+            alt="searchIcon"
+            onClick={methods.handleSubmit((data) => {
+              console.log(data);
```
Comment (lyudad) [inline src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/index.tsx]: 1.1.8

File: src/pages/Orders/api/getOrders.ts [hunk 3]
```diff
@@ -20,12 +22,20 @@ export const getOrders = async ({
   search,
   page,
   companyId,
+  isNew,
 }: GetOrdersParams): Promise<void> => {
   try {
+    store.dispatch(
+      setAccessToken({
+        token:
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MiwibG9nbyI6Imh0dHA6Ly9sb2dvIiwiZnVsbF9uYW1lIjoiQWRtaW4iLCJlbWFpbCI6ImFzZEBnbWFpbC5jb20iLCJyb2xlIjoiYWRtaW4iLCJjb21wYW55X2lkIjp7ImlkIjoyLCJuYW1lIjoiTHVnZ2FnZSBkZWxpdmVyeSIsImxvZ28iOiJodHRwczovL3d3dy5nb29nbGUuY29tL3VybD9zYT1pJnVybD1odHRwcyUzQSUyRiUyRmxvZ28uY29tJTJGbG9nb3MlMkZkZWxpdmVyeSZwc2lnPUFPdlZhdzFyYzM5ZklOUUJsYjdZRGZYTDBGSmsmdXN0PTE3MzA1Mzk5NzA2NzIwMDAmc291cmNlPWltYWdlcyZjZD12ZmUmb3BpPTg5OTc4NDQ5JnZlZD0wQ0JRUWpSeHFGd29UQ01qUHU0VHF1b2tERlFBQUFBQWRBQUFBQUJBRSIsImVtYWlsIjoiZGVsaXZlcnkuY29tcGFueUBnbWlhbC5jb20iLCJjcmVhdGVkQXQiOiIyMDI0LTExLTE0VDA5OjQzOjUyLjMwOFoiLCJ1cGRhdGVkQXQiOiIyMDI0LTExLTE0VDA5OjQzOjUyLjMwOFoifSwiY3JlYXRlZEF0IjoiMjAyNC0xMS0xNFQwOTo0NDoxMy41NzRaIiwidXBkYXRlZEF0IjoiMjAyNC0xMS0xNFQwOTo0NDoxMy41NzRaIiwiaWF0IjoxNzMxNTg0NjUzfQ.8xGQt2L7LmoVWECMF55kLlMzTNz6uhoz6SXYg_kzZNY',
```
Comment (lyudad) [inline src/pages/Orders/api/getOrders.ts]: неожиданно. зачем это?

File: src/pages/CreateRoute/SecondStage/components/OrderManagementCard/styles.ts (hunk 1 above)
Comment (Parle648) [reply src/pages/CreateRoute/SecondStage/components/OrderManagementCard/styles.ts]: replaced by reusable font

File: src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/index.tsx (hunk 2 above)
Comment (Parle648) [reply src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/index.tsx]: delete useless console.log

File: src/pages/Orders/api/getOrders.ts (hunk 3 above)
Comment (Parle648) [reply src/pages/Orders/api/getOrders.ts]: deleted

File: src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/index.tsx
Comment (lyudad) [inline src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/index.tsx:1]: а зачем нам создавать каждый раз компонент под серч, они же одинаковые. мне кажется он есть в компонентах, если нет, то вынести сам компонент туда. и компонент будет принимать фунркцию отправки на сервер

File: src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/useSearch.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+type UseSearchType = () => { sendRequestByParams: (search: string) => void };
+
+export const useSearchOrders: UseSearchType = () => {
+  const { params } = useSelector((store: RootState) => store.ordersPageSlice);
+
+  const sendRequestByParams = (search: string) => {
+    store.dispatch(setSearchBy(search));
+
+    getOrders({
```
Comment (lyudad) [inline src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/useSearch.ts]: нужен debounce, отправлять запрос на сервер без задержки не хорошая практика

File: src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/index.tsx
Comment (Parle648) [reply src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/index.tsx:1]: create reusable search component

File: src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/useSearch.ts (hunk 4 above)
Comment (Parle648) [reply src/pages/CreateRoute/SecondStage/components/OrdersManagementSearch/useSearch.ts]: apply debounce

File: src/components/SearchComponent/index.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+import searchIcon from '@/assets/icons/search.svg';
+import { rowStyles, formWrapper, searchInputStyles } from './styles';
+import BasicDatePicker from '@/components/BasicDatePicker';
+
+const SearchComponent: FC<{ onSearch: (search: string) => void }> = ({
+  onSearch,
+}) => {
+  const methods = useForm<{ search: string }>();
+  //   const { sendRequestByParams } = useSearchOrders();
```
Comment (lyudad) [inline src/components/SearchComponent/index.tsx]: 1.1.10

File: src/components/SearchComponent/index.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+  // sendRequestByParams(data.search)
```
Comment (lyudad) [inline src/components/SearchComponent/index.tsx]: 1.1.10

File: src/components/SearchComponent/index.tsx (hunk 5 above)
Comment (Parle648) [reply src/components/SearchComponent/index.tsx]: fix

File: src/components/SearchComponent/index.tsx (hunk 6 above)
Comment (Parle648) [reply src/components/SearchComponent/index.tsx]: fix

**Pull request merged**